src/app/code/community/Omise/Gateway/Model/Order.php

```diff
@@ -15,6 +15,16 @@ public function getOrder($id = null)
         return $this->loadBySession();
     }
 
+    /**
+     * Check whether the payment is in payment pending state
```

Comment:
I might be the last person to speak about it.. but maybe comment here would be better:
`Check whether the payment is in pending state`

that is like `polish language` thing.. that we should avoid repeating words if possible..

Reply:
Either is correct really - I simply copied this from the Magento code and changed 'review' to 'pending'. English style rules generally frown upon the use of repeated words too though.